src/main/java/org/jitsi/jibri/JibriManager.kt

```diff
@@ -0,0 +1,134 @@
+package org.jitsi.jibri
+
+import org.jitsi.jibri.config.JibriConfig
+import org.jitsi.jibri.health.JibriHealth
+import org.jitsi.jibri.service.*
+import org.jitsi.jibri.service.impl.FileRecordingJibriService
+import org.jitsi.jibri.service.impl.StreamingJibriService
+import org.jitsi.jibri.service.impl.RecordingOptions
+import org.jitsi.jibri.service.impl.StreamingOptions
+import org.jitsi.jibri.util.error
+import java.io.File
+import java.util.logging.Logger
+
+enum class StartServiceResult {
+    SUCCESS,
+    BUSY,
+    ERROR
+}
+
+data class FileRecordingParams(
```

Comment:
Maybe there's an argument for this style but I would normally just pass the params in directly instead of via a data class

Reply:
i may do this...for some reason having them tucked away in a params class to start out (as i was discovering what needed to be passed in) felt better for some reason.  now that things are more settled maybe i'll take a look at changing it.